### PR TITLE
Cleanup double initialized fields in SPC

### PR DESCRIPTION
**Pull request by marek-safar**

### CODE REVIEW

File: src/libraries/System.Private.CoreLib/src/System/AttributeUsageAttribute.cs [hunk 1]
```diff
@@ -18,14 +18,15 @@ namespace System
     public sealed class AttributeUsageAttribute : Attribute
     {
         private readonly AttributeTargets _attributeTarget; // Defaults to all
```
Comment (jkotas) [inline src/libraries/System.Private.CoreLib/src/System/AttributeUsageAttribute.cs]:
```suggestion
        private readonly AttributeTargets _attributeTarget;
```

File: src/libraries/System.Private.CoreLib/src/System/AttributeUsageAttribute.cs [hunk 2]
```diff
@@ -18,14 +18,15 @@ namespace System
...
-        private bool _allowMultiple = false; // Defaults to false
-        private bool _inherited = true; // Defaults to true
+        private bool _allowMultiple;
+        private bool _inherited;
 
         internal static readonly AttributeUsageAttribute Default = new AttributeUsageAttribute(AttributeTargets.All);
```
Comment (jkotas) [inline src/libraries/System.Private.CoreLib/src/System/AttributeUsageAttribute.cs]: You can also change this to `const` while you are on it to get rid of the static constructor.

File: src/libraries/System.Private.CoreLib/src/System/Version.cs [hunk 3]
```diff
@@ -85,8 +88,8 @@ public Version(string version)
 
         public Version()
         {
-            _Major = 0;
```
Comment (jkotas) [inline src/libraries/System.Private.CoreLib/src/System/Version.cs]: For clarity, I would keep `_Major = 0;` and `_Minor = 0;` here, but comment them out.

**Review by jkotas — APPROVED** [review]

File: src/libraries/System.Private.CoreLib/src/System/AttributeUsageAttribute.cs (hunk 2 above)
Comment (marek-safar) [reply src/libraries/System.Private.CoreLib/src/System/AttributeUsageAttribute.cs]: I cannot as `AttributeUsageAttribute` is a class

**Review by stephentoub — APPROVED** [review]

**Pull request merged**